src/app/tab1/tab1.page.ts

```diff
@@ -1,8 +1,15 @@
-import { Component } from '@angular/core';
-
+import { PlanningComponent } from '../operations/planning/planning.component';
+import { Component, ViewChild } from '@angular/core';
 @Component({
   selector: 'app-tab1',
   templateUrl: 'tab1.page.html',
   styleUrls: ['tab1.page.scss']
 })
-export class Tab1Page {}
+export class Tab1Page {
+  @ViewChild('appPlanning')
+  private test: PlanningComponent ;
+
+  ionViewDidEnter() {
```

Comment:
Missing JSDoc

Reply:
We don't put JSDocs on default Angular/Ionic lifecycle functions